docs/workflow-job-submission.md

```diff
@@ -109,57 +99,61 @@ dbt will generate a name based on the catalog, schema, and model identifier.
+- Similarly, you can define a reusable job cluster for the workflow and tell the task to use that
 - If none of those are in the configuration, the task cluster will be serverless
 
 ```yaml
 # Reusable job cluster config example
 
 models:
   - name: my_model
-    
+
     config:
-      workflow_job_config:
-        additional_task_settings: {
-          task_key: 'task_a',
-          job_cluster_key: 'cluster_a',
-        }
-        post_hook_tasks: [{
-          depends_on: [{ "task_key": "task_a" }],
-          task_key: 'OPTIMIZE_AND_VACUUM',
-          job_cluster_key: 'cluster_a',
-          notebook_task: {
-            notebook_path: "/OPTIMIZE_AND_VACUUM",
-            source: "WORKSPACE",
-            base_parameters: {
-              database: "{{ target.database }}",
-              schema: "{{ target.schema }}",
-              table_name: "my_model"
-            }
-          },
-        }]
-        job_clusters: [{
-          job_cluster_key: 'cluster_a',
-          new_cluster: {
-            spark_version: "{{ var('dbr_versions')['lts_v14'] }}",
-            node_type_id: "{{ var('cluster_node_types')['large_job'] }}",
-            runtime_engine: "{{ var('job_cluster_defaults.runtime_engine') }}",
-            autoscale: {
-              "min_workers": 1,
-              "max_workers": 2
+      python_job_config:
+        additional_task_settings:
+          { task_key: "task_a", job_cluster_key: "cluster_a" }
+        post_hook_tasks:
+          [
+            {
+              depends_on: [{ "task_key": "task_a" }],
+              task_key: "OPTIMIZE_AND_VACUUM",
+              job_cluster_key: "cluster_a",
+              notebook_task:
+                {
+                  notebook_path: "/OPTIMIZE_AND_VACUUM",
+                  source: "WORKSPACE",
+                  base_parameters:
+                    {
+                      database: "{{ target.database }}",
+                      schema: "{{ target.schema }}",
+                      table_name: "my_model",
+                    },
+                },
             },
-          }
-        }]
+          ]
+        job_clusters:
+          [
+            {
+              job_cluster_key: "cluster_a",
+              new_cluster:
+                {
+                  spark_version: "{{ var('dbr_versions')['lts_v14'] }}",
+                  node_type_id: "{{ var('cluster_node_types')['large_job'] }}",
+                  runtime_engine: "{{ var('job_cluster_defaults.runtime_engine') }}",
+                  autoscale: { "min_workers": 1, "max_workers": 2 },
+                },
+            },
+          ]
 ```
 
 #### Grants
 
 You might want to give certain users or teams access to run your workflows outside of
-dbt in an ad hoc way. You can define those permissions in the `workflow_job_config.grants`.
+dbt in an ad hoc way. You can define those permissions in the `python_job_config.grants`.
```

Comment:
I changed this with find/replace, and VS decided it needed to reformat the whole file :/